bundles/org.openhab.binding.mystrom/src/main/java/org/openhab/binding/mystrom/internal/MyStromHandlerFactory.java

```diff
@@ -61,6 +65,10 @@ public boolean supportsThingType(ThingTypeUID thingTypeUID) {
             return new MyStromHandler(thing, httpClientFactory.getCommonHttpClient());
         }
 
+        if (THING_TYPE_BULB.equals(thingTypeUID)) {
```

Comment:
```suggestion
        } else if (THING_TYPE_BULB.equals(thingTypeUID)) {
```

Reply:
Is that `else` not superfluous. I think this is very controversial on the net. I don't care. I can insert `else`.